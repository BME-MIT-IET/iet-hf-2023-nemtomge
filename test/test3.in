loadmap map.json 
start V1 V2 V3
set Scientist V1 PL1
set Scientist V2 PL2
set Scientist V3 PL2
show Map 
move V1 PL2 
show Map
exit
