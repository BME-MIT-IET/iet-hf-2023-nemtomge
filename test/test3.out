Map from file map.json loaded
The game has been started. Players are: V1 V2 V3
Scientist V1 set on field PL1
Scientist V2 set on field PL2
Scientist V3 set on field PL2
The map consists of these non-empty fields:
Plain(PL1): 
 - V1
Plain(PL2):
 - V2
 - V3
Scientist V1 couldn't move
The map consists of these non-empty fields: 
Plain(PL1): 
 - V1
Plain(PL2):
 - V2
 - V3
